Runtime/Helpers/CinemachineDefaultMouseInput.cs

```diff
@@ -0,0 +1,75 @@
+#if CINEMACHINE_UNITY_INPUTSYSTEM
+
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Cinemachine
+{
+    /// <summary>
+    /// Static class that returns a reference to InputSystem package's PlayerLook input asset.
+    /// </summary>
+    static class CinemachineDefaultMouseInput
```

Comment:
Why is this class in the runtime assembly instead of the editor?  It's editor stuff.